src/Api/fetchCitiesToSearcher.tsx

```diff
@@ -0,0 +1,21 @@
+const fetchDataCity = async (url) => {
+	const response = await fetch(url, {
+		headers: {
+			"X-Api-Key": "ltR5wJ+f1gVCJhhbP5N41g==VusBG342pBT6r6cm",
```

Comment:
api-key is a sensitive data. You never want to publish that kind of information. You need to place in your `.env` file that should be [git ignored](https://git-scm.com/docs/gitignore)
```suggestion
			"X-Api-Key": process.env.NINJAS_API_KEY
```